### PR TITLE
Features/ondemand email frontend

### PR DESCRIPTION
**Pull request by TeunMos**

### CODE REVIEW

File: kdp-frontend/src/views/rockstar/store/rockstars.ts [hunk 1]
```diff
@@ -71,6 +72,10 @@ const rockstar = {
   },
   mutations: {
     SET_ROCKSTAR: (state: rockstarState, data: RockstarShape) => {
+      console.log(data);
+
+      console.log(data);
```
Comment (ThijmenBrand) [inline kdp-frontend/src/views/rockstar/store/rockstars.ts]: Remove

File: kdp-frontend/src/views/rockstar/store/rockstars.ts [hunk 2]
```diff
@@ -71,6 +72,10 @@ const rockstar = {
   },
   mutations: {
     SET_ROCKSTAR: (state: rockstarState, data: RockstarShape) => {
+      console.log(data);
```
Comment (ThijmenBrand) [inline kdp-frontend/src/views/rockstar/store/rockstars.ts]: Remove

File: kdp-frontend/src/components/rockstarView/RockstarView.vue [hunk 3]
```diff
@@ -65,6 +65,11 @@
             />
           </a>
         </div>
+
+        <div v-if="FormSucces">
+            <p class="email-succes">OnDemand request succesfully send!</p>
```
Comment (ThijmenBrand) [inline kdp-frontend/src/components/rockstarView/RockstarView.vue:70]: Add language support

File: kdp-frontend/src/components/rockstarView/RockstarView.vue [hunk 4]
```diff
@@ -167,6 +177,9 @@
 import { RockstarShape } from "@/models/Rockstar";
 import { reactive, ref } from "vue";
 import Modal from "@/components/modal/Modal.vue";
+import rockstarService from "@/services/callFunctions/rockstar";
+import  RockstarOnDemandRequest from "@/models/RockstarOnDemandRequest";
```
Comment (ThijmenBrand) [inline kdp-frontend/src/components/rockstarView/RockstarView.vue:181]: two spaces?

**Review by RenoMuijsenberg — APPROVED** [review]

**Pull request merged**